source/08-sushi/01-counter-sushi-api-paths-to-support.rst

```diff
@@ -34,7 +34,7 @@ The following paths (methods) MUST be supported:
 
    * - GET
      - /r51/reports
-     - Returns a list of reports supported by the COUNTER_SUSHI API service. The response includes an array of reports, including the report identifier, the release number, the report name, a description, and (optional but recommended for custom reports) the path to use when requesting the report.
+     - Returns a list of reports supported by the COUNTER_SUSHI API service. The response includes an array of reports, including the report identifier, the release number, the report name, a description, the path to use when requesting the report (optional but recommended for custom reports), and the first and last months for which usage data has been processed and is available. If possible, this information should be customer specific.
```

Comment:
My impression is that many report providers won't be able to provide customer specific information. The SHOULD seems to indicate that the report provider should make an effort to provide customer specific information, or that they would have to contact you if they aren't able to provide customer specific information to resolve that conflict. If that's the intention then we indeed should make that change (with customer ID instead of Customer_ID for consistency).